layout

```diff
@@ -0,0 +1,244 @@
+        top: Reproducibility
+        top: Scheduling
+        top: System Architecture
+        top: Determined on AWS
+        top: Determined on GCP
+        top: Elastic Infrastructure
+        top: Dynamic Agents on AWS
+        top: Dynamic Agents on GCP
+        top: Terminology and Concepts
+X       top: User Interfaces
+        top: Users
+X       top: YAML Topic Guide
+
+    Reference
+        ref: API Reference
+        ref: Cluster Configuration
+        ref: Command-line Interface
+        ref: Tensorboard and Notebook Configuration
+        ref: Configuration Templates
+        ref: Experiment Configuration
+        ref: Helm Chart Configuration
+        ref: Open Source Licenses
+
+    Frequently Asked Questions
+    Examples
+    Release Notes
+
+
+####### PROPOSED REORG
+
+# Guiding principles of reorg:
+#  - topical organization
+#  - fewer pages = better ctrl-f and less hierarchy to navigate
+
+N=new content   *=doc-level item   ?=not sure if we have this already
+
+    Quick Start
+N     * Running Your First Experiment
+            prereqs:
+                docker works
+            goals:
+                have working installation (docker method)
+                have cli installed
+                    how: Install Determined CLI
+                hands-on use of cli and webui (tensorboard?), and nothing else
+            details:
+                use one-variable model (copy/paste rather than download code)
+N     * Porting A PyTorch Model
+            prereqs:
+                have a working installation
+            steps:
+                start with a one-variable model  (use mnist instead?)
+                show a DummyContext with a skeleton training loop
+                implement build_model + train_batch
+                implement build_*_data_loader
+                run in local test mode
+                run in remote test mode
+                run a full experiment
+            (is that too many steps?)
+N     * Porting A Keras Model
+            prereqs:
+                have a working installation
+            steps:
+                start with a one-variable model  (use mnist instead?)
+                show a DummyContext with a skeleton training loop
+                implement build_model
+                implement build_*_data_loader
+                run in local test mode
+                run in remote test mode
+                run a full experiment
+            (is that too many steps?)
+  * Concepts
+        top: Benefits of Determined  (cannabalize this)
+        Training Models
+            top: Model Definitions
+            top: Experiment Lifecycle
+        Hyperparameter Tuning
+        Notebooks
+        Tensorboard
+        Model Management
+        System Components (basically a glossary)
+            top: System Architecture
+            top: Terminology and Concepts
+  * Training Models
+        Model APIs  (what should go here, other than links to referece docs?)
```

Comment:
I don't think we need a ton of content here, but might be worth covering

* Why we need a trial API in the first place
* Concepts that apply to all of the trial APIs (e.g., trial context)
* Concepts that apply to each specific API in turn (e.g., how/why the PyTorch API works the way it does vs. how the Keras API works).

Reply:
Stealing this for the RFC